src/site/markdown/server/manage-webadmin.md

```diff
@@ -238,7 +238,23 @@ curl -XGET http://ip:port/quota/users/usernameToBeUsed
 
 Resource name usernameToBeUsed should be an existing user
 
-The answer can contain a fixed value, an empty value (null) or an unlimited value (-1):
+The answer is the details of the quota of that user.
+
+```
+{
+  "global": {"count":252,"size":242}
```

Comment:
comma is missing at the end of the line (also on others)

Reply:
Well  spotted. I'll correct it.